src/api_server/src/lib.rs

```diff
@@ -137,15 +138,25 @@ impl ApiServer {
             match server.requests() {
                 Ok(request_vec) => {
                     for server_request in request_vec {
+                        let request_processing_start_us =
+                            utils::time::get_time_us(utils::time::ClockType::Monotonic);
                         server
                             .respond(
                                 // Use `self.handle_request()` as the processing callback.
-                                server_request.process(|request| self.handle_request(request)),
+                                server_request.process(|request| {
+                                    self.handle_request(request, request_processing_start_us)
+                                }),
                             )
                             .or_else(|e| {
                                 error!("API Server encountered an error on response: {}", e);
                                 Ok(())
                             })?;
+                        let delta_us = utils::time::get_time_us(utils::time::ClockType::Monotonic)
+                            - request_processing_start_us;
+                        debug!(
+                            "Handling the request from the previous API call, including waiting for the VMM response, took {} us.",
```

Comment:
Please use a short message like `Total API call time: %u`.
All the long static strings throughout the codebase add up and bloat the binary size.

Reply:
Done.